policyengine_us/variables/gov/states/sc/tax/income/credits/college_tuition/sc_tuition_credit_eligible.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class sc_tuition_credit_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible for the South Carolina Tuition Credit"
+    definition_period = YEAR
+    reference = (
+        "https://dor.sc.gov/forms-site/Forms/I319_2021.pdf#page=2",
+        "https://www.scstatehouse.gov/code/t12c006.php"
+        # South Carolina Legal Code | SECTION 12-6-3385 (B)(3)(b)
+    )
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.sc.tax.income.credits.college_tuition
+        total_college_hours = person("total_college_hours", period)
+        num_of_semester_attended = person("college_semesters_attended", period)
+        return where(
+            total_college_hours == 0,
+            0,
+            total_college_hours
```

Comment:
cant this be the condition?          
```
total_college_hours
>= p.semester_hour_requirement
```
separate this statement out into a separate line